mod_sample/controllers.py

```diff
@@ -30,11 +32,27 @@ def before_app_request():
 
 
 class SampleNotFoundException(Exception):
+
     def __init__(self, message):
         Exception.__init__(self)
         self.message = message
 
 
+def list_github_issue(label):
+    from run import config
+    REPO_OWNER = config.get('GITHUB_OWNER', '')
+    REPO_NAME = config.get('GITHUB_REPOSITORY', '')
+    url = 'https://api.github.com/search/issues?q=+label:%s+'
+    'repo:%s+user:%s' % (
```

Comment:
https://api.github.com/search/issues?q=+label:sample1+repo:satyammittal/library yields results too, so no reason to add the user, unless there's a reason for that?